README.md

```diff
@@ -606,12 +606,14 @@ type config struct {
 
 var C config
 
-err := Unmarshal(&C)
+err := viper.Unmarshal(&C)
 if err != nil {
 	t.Fatalf("unable to decode into struct, %v", err)
 }
 ```
 
+`mapstructure` is the only supported tag.
```

Comment:
Maybe say: Viper uses [github.com/mitchellh/mapstructure](https://github.com/mitchellh/mapstructure) under the hood for unmarshaling values which (similarly to `encoding/json`) uses `mapstructure` tags by default.